fcrepo-import/src/main/java/org/fcrepo/importer/Importer.java

```diff
@@ -83,14 +87,10 @@ private FcrepoClient client() {
      */
     public void run() {
         logger.info("Running importer...");
-        try {
```

Comment:
They're already caught and logged by the driver.

https://github.com/fcrepo4-labs/fcrepo-import-export/blob/master/fcrepo-import-export-driver/src/main/java/org/fcrepo/importexport/ImportExportDriver.java#L47-L50